django/db/migrations/executor.py

```diff
@@ -86,6 +86,11 @@ def migrate(self, targets, plan=None, state=None, fake=False, fake_initial=False
         Django first needs to create all project states before a migration is
         (un)applied and in a second step run all the database operations.
         """
+
+        # Make sure the migrations table is there. We don't want to fail with
```

Comment:
suggested wording :`# The django_migrations table must be present to record applied migrations.`